File: Algorithms/Python/Fibonacci.py
import sys

def Fibonacci(iterationsToDo):
	previous = 0
	secondPrevious = 1

	for iteration in range(iterationsToDo):
		aux = previous
		previous += secondPrevious
		secondPrevious = aux
		print(previous, " ")

Fibonacci(int(sys.argv[1]))

File: Algorithms/Java/Fibonacci.java
public class Fibonacci {

	public static void main(String args[]){
		int iterationsToDo = Integer.parseInt(args[0]);

		int previous = 0;
		int secondPrevious = 1;

		for (int iteration = 0; iteration < iterationsToDo; iteration++){
			int aux = previous;
			previous += secondPrevious;
			secondPrevious = aux;
			System.out.print(current + " ");
		}
	}

}